core-api/src/test/java/com/sequenceiq/cloudbreak/api/model/imagecatalog/ValidatorTestHelper.java

```diff
@@ -0,0 +1,38 @@
+package com.sequenceiq.cloudbreak.api.model.imagecatalog;
+
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import javax.validation.ConstraintViolation;
+import javax.validation.Validator;
+
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = { ValidatorTestHelperConfiguration.class })
+public class ValidatorTestHelper {
+
+    @Autowired
+    private Validator validator;
+
+    protected List<String> getPropertyPaths(Set<? extends ConstraintViolation<?>> violations) {
+        return violations.stream().map(ConstraintViolation::getPropertyPath).map(Object::toString).collect(Collectors.toList());
+    }
+
+    protected List<String> getMessageTemplate(Set<? extends ConstraintViolation<?>> violations) {
+        return violations.stream().map(ConstraintViolation::getMessageTemplate).map(msg -> msg.replaceAll("([{}])", ""))
+                .collect(Collectors.toList());
+    }
+
+    public Validator getValidator() {
+        return validator;
+    }
+
+    public void setValidator(Validator validator) {
+        this.validator = validator;
+    }
```

Comment:
Is this needed?

Reply:
I had to add it because of a checkstyle error. Should I suppress it?

Reply:
Nevermind